src/srcassets/Home.vue

```diff
@@ -0,0 +1,103 @@
+<template>
```

Comment:
The file path here is throwing me off, Do we have a home component in our srcassets folder?